python/revng/pypeline/cli/project/analyze.py

```diff
@@ -121,7 +121,7 @@ def _build_analysis_list_command(self, analysis_list_name: str, pipeline: Pipeli
             config = f'Configuration for the analysis list "{analysis_list_name}".'
             run_analysis_command = click.option(
                 f"--{normalize_flag(analysis_name)}-configuration",
-                f"{analysis_name}_configuration",
+                f"{normalize_pos_arg_name(analysis_name)}_configuration",
```

Comment:
The CLI arguments use `-`, not `_`, correct?

Reply:
Yes, but the variable where this is stored needs to be snake_case